drivers/flash/flash_stm32g4x.c

```diff
@@ -101,11 +94,12 @@ static int write_dword(const struct device *dev, off_t offset, uint64_t val)
 	return rc;
 }
 
-static int erase_page(const struct device *dev, unsigned int page)
+static int erase_page_offset(const struct device *dev, unsigned int offset)
```

Comment:
I don't think the function name change was required as I am getting confused with the _offset suffix.
IMO, the change of parameter name was sufficient to notice the change if the function behavior.

Reply:
Done